docs/ocm-integration.md

```diff
@@ -242,3 +242,149 @@ ocm get subscriptions --parameter search="status = 'Active'"
+
+```shell
+$ ocm whoami | jq -r '.organization.id'
+2GOuuT2l3odeHRdWqY57fviSE2K
+```
+
+And for a given user name:
+
+```shell
+$ ocm get /api/accounts_mgmt/v1/accounts -p search="username = 'jane.doe'" | jq -r '.items[].organization.id'
+2GOuxr72XcFbrWvyA9rS9McOUjO
+```
+
+Then add the label to the organization (labels are the mechanism used by AMS to
+store these capabilities):
+
+```shell
+$ ocm post /api/accounts_mgmt/v1/organizations/2GOuxr72XcFbrWvyA9rS9McOUjO/labels <<.
+{
+  "key": "capability.organization.bare_metal_installer_multiarch",
+  "value": "true",
+  "internal": true
+}
+.
+```
+
+To disable a capability just delete the label:
+
+```shell
+$ ocm delete /api/accounts_mgmt/v1/organizations/2GOuxr72XcFbrWvyA9rS9McOUjO/labels/capability.organization.bare_metal_installer_multiarch
+```
+
+To check the capabilities enabled for an organization:
+
+```shell
+$ ocm get /api/accounts_mgmt/v1/organizations/2GOuxr72XcFbrWvyA9rS9McOUjO/labels | jq -r '.items[] | select(.type == "Organization") | .key'
+sts_ocm_role
+capability.organization.bare_metal_installer_multiarch
+capability.organization.hypershift
+capability.cluster.subscribed_ocp
+moa_entitlement_expires_at
+```
+
+Chances are that you will not have permission to enable or disable
+capabilities, specially in the production environment. In that case open a
+ticket like [this](https://issues.redhat.com/browse/SDB-3211) in the
+[SDB](https://issues.redhat.com/projects/SDB) board requesting it. Make sure to
+include in the ticket the name of the capability that you want to enable, the
+identifier of the organization and the environment. For example, you could can
+write a title and description like these:
+
+```
+Enable `bare_metal_installer_multiarch` capability for `My Org` in production
+
+We need to enable the `bare_metal_installer_multiarch` capability for the
+`2GOuxr72XcFbrWvyA9rS9McOUjO` organization (also known as `My Org`) in the
+production environment. This is necessary to enable multiarch support in the
+assisted installer service.
+
+Adding the capability should be something like this:
+
+{noformat}
+$ ocm login --token ... --url production
+$ ocm post /api/accounts_mgmt/v1/organizations/2GOuxr72XcFbrWvyA9rS9McOUjO/labels <<.
+{
+  "key": "capability.organization.bare_metal_installer_multiarch",
+  "value": "true",
+  "internal": true
+}
+{noformat}
+```
+
+Note that organization identifiers and capabilities are environment specific,
+so make sure you do it in the right environment. For example, to use the
+integration environment:
+
+```shell
+$ ocm login --token ... --url integration
+```
+
+To use the stage environment:
+
+```shell
+$ ocm login --token ... --url stage
+```
+
+The default, when the `--url` option isn't provided is to use the production
+environment.
+
+If you need to do this very often in the integration or stage environments, and
+without asking the SDB team, you can request the `BareMetalInstallerAdmin` role
```

Comment:
add ``- UHCSupport:`` and it's a complete solution
(https://gitlab.cee.redhat.com/service/ocm-resources/-/merge_requests/2725)

Reply:
Done.